Dockerfile

```diff
@@ -50,12 +50,11 @@ RUN apk --no-cache add --virtual \
 
 WORKDIR /app
 
-COPY --from=build /app/node_modules /app/node_modules
-COPY --from=build /app/.next /app/.next
 COPY --from=build /app/next.config.js /app/next.config.js
 COPY --from=build /app/public /app/public
 COPY --from=build /app/package.json /app/package.json
-COPY --from=build /app/yarn.lock /app/yarn.lock
+COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
```

Comment:
Why is `chown` used here but not for the other COPY directives? Is it needed here?

Reply:
Also, can we specify the full destination path like the other directives? (`./` vs `/app/.....`)

Reply:
I've tested again, `--chown` is not required, so I will be removing this.  Also, changed the path to the full path.